src/Controls/src/SourceGen/InitializeComponentCodeWriter.cs

```diff
@@ -87,21 +87,35 @@ PrePost newblock() =>
 				codeWriter.WriteLine($"private partial void {methodName}()");
 				xamlItem.Root!.XmlType.TryResolveTypeSymbol(null, compilation, xmlnsCache, out var baseType);
 				var sgcontext = new SourceGenContext(codeWriter, compilation, sourceProductionContext, xmlnsCache, typeCache, rootType!, baseType, xamlItem.ProjectItem);
-				using (newblock())
+				if (xamlItem.ProjectItem.TreeOrder)
```

Comment:
I'm a little confused. Why do we want to keep the previous way of generating code around? Don't we want to fully replace it with the new codegen? Is this just temporary until we fix all the existing unit tests?